views/layouts/main.handlebars

```diff
@@ -13,10 +13,17 @@
   <script defer src="/l20n/dist/web/l20n.js"></script>
 </head>
 <body>
-  <div class="send-logo">
-    <img src="/resources/send_logo.svg"/>
-    <img src="/resources/send_logo_type.svg"/>
-  </div>
+  <header class="header"> 
+    <div class="send-logo">
+      <img src="/resources/send_logo.svg"/>
```

Comment:
Not sure if we want an `alt` attr here, or some `data-l10n-id` (although I don't suspect this `alt=""` would need to be translated if it is just "Firefox Send".